src/pages/mining.md

```diff
@@ -126,6 +126,108 @@ In case you are running into issues or would like to see verbose logging, you ca
 BLOCKSTACK_DEBUG=1 stacks-node krypton
 ```
 
+## Running a miner in Windows
+
+### Generate keychain and get testnet tokens in Windows
+
+To setup the miner, first, we need to generate a keychain. With this keychain, we'll get some testnet BTC from a faucet, and then use that BTC to start mining.
+
+To get a keychain, the simplest way is to use the `stacks-cli`.We'll use the `stx make-keychain` command,and pass -t to indicate that we want a testnet keychain.
+
+Generate a keychain:
+
+```bash
+npm install --global @stacks/cli
+stx make_keychain -t > cli_keychain.json
+type cli_keychain.json
```

Comment:
so, if I read this correctly, this is the most divergence from the non-windows steps, correct? if it is just that - we should rather make a note in the other tutorial and avoid duplicating 90% of content from the other tutorial. wdyt @HariniRajan397 ?